client/reader/discover/discover-stream.js

```diff
@@ -23,6 +23,33 @@ import {
 	FIRST_POSTS_TAB,
 } from './helper';
 
+export const DiscoverHeader = ( props ) => {
+	const translate = useTranslate();
+
+	const { selectedTab } = props;
+	const tabTitle = getSelectedTabTitle( selectedTab );
+	let subHeaderText = translate( 'Explore %s blogs that inspire, educate, and entertain.', {
+		args: [ tabTitle ],
+		comment: '%s is the type of blog being explored e.g. food, art, technology etc.',
+	} );
+	if ( selectedTab === FIRST_POSTS_TAB ) {
+		subHeaderText = translate(
+			'Fresh voices, fresh views. Explore first-time posts from new bloggers.'
+		);
+	}
+
+	return (
+		<NavigationHeader
+			navigationItems={ [] }
```

Comment:
Pre-existing, but while we're here: I'd suggest that we declare this array outside the component as a constant, and always pass the same array to `navigationItems`.

```suggestion
			navigationItems={ [] }
```

Reply:
Good point, thanks! I've moved it outside the component as a const.